src/lib/components/workflow/workflow-reset-form.svelte

```diff
@@ -46,36 +29,36 @@
       label: 'None',
     },
   ];
-
-  const handleResetIdInput = (event: InputEvent) => {
-    const eventTarget = event.target as HTMLInputElement;
-    if (eventTarget.value) {
-      const parsed = parseInt(eventTarget.value);
-      const lastEventId = lastEvent?.id;
-      if (parsed < 1 || parsed > parseInt(lastEventId)) {
-        eventIdValid = false;
-        resetIdHintText = `Please enter an Event ID between 1 and ${lastEventId}`;
-      } else {
-        eventIdValid = true;
-        resetIdHintText = DEFAULT_RESET_ID_HINT_TEXT;
-      }
-    }
-  };
 </script>
 
 <div class="flex w-full flex-col gap-4">
-  <Select
-    label="Reset to"
-    id="reset-type-select"
-    bind:value={resetType}
-    testId="workflow-reset-type-select"
-  >
-    {#each resetTypes as { value, label }}
-      <Option {value}>
-        {label}
-      </Option>
+  <ul class="w-full rounded border border-primary max-h-40 overflow-y-scroll">
+    {#each $resetEvents as event}
+      <li
+        class="w-full h-10 border-b border-primary last-of-type:border-none hover:bg-gradient-to-br from-blue-100 to-purple-100 bg-fixed first-of-type:rounded-t last-of-type:rounded-b"
+      >
+        <label
```

Comment:
I love the wrapping an `<input>` in a `<label>` tag as much as the next person, but we should just keep them in parallel and use the `for` attribute and an `id` on the input.

Reply:
https://www.w3.org/TR/2008/WD-WCAG20-TECHS-20080430/H44.html

> The HTML and XHTML specifications allow both implicit and explicit labels. However, some assistive technologies do not correctly handle implicit labels

I think maybe the solution here is keep the wrapping label but also add a `for` attr

Reply:
Yea, and with the `for` attribute, you also get the ability to click on the label—that's what one me over eventually.